### DISCUSSION TITLE
Example CodeSandbox links don't work?

### DISCUSSION THREAD

**Discussion by davidburson** [igordanchenko/react-photo-album#117] [Q&A] [answered]
I'd like to play with the [Rows example](https://react-photo-album.com/examples/rows), but when I click the CodeSandbox button, CodeSandbox reports that none of the files exist (although it shows the project running).  The error it shows is:  `Error:  Unable to read file '\project\sandbox\src\App.tsx' (Error: No such file or directory)`

I tried 2 different browsers (on Windows:  Chrome and Edge) and rebooted - not sure why CodeSandbox thinks the files don't exist.  I think this was working last week (though I'm not positive about that, might have been a different library I was looking at).

Any ideas?

![Screenshot 2023-07-12 121647](https://github.com/igordanchenko/react-photo-album/assets/6555661/47cefc81-da94-4d78-b4a4-d7bddeaa21b4)

**Comment by igordanchenko** [comment]
It must be a CodeSandbox glitch. It looks like it replaces `/` path separator with `\` because you are on Windows, and is unable to resolve that path. I'm unable to reproduce this issue myself as I'm on Mac.

Have you tried forking the sandbox? Have you tried closing the editor tab and re-opening it from the left-hand side navigation menu?

And, of course, you always have an option to clone this [repository](https://github.com/igordanchenko/react-photo-album/tree/main) and run the examples locally.

**Reply by davidburson** [reply]
Thanks, I tried closing the editor tab and re-opening from the left-hand side navigation menu.  No file will open.  I get the same error (except the error includes filename I try to open, of course).  I also tried forking.  Same problem in my forked version.

I tried on a Mac, and it works fine, as you also saw.

There seems to be something different about this link as compared to other CodeSandbox links.  Other links work fine on Windows.  For example, in [this comment](https://github.com/igordanchenko/react-photo-album/discussions/67#discussioncomment-4561261) you created this link, which works fine:  https://codesandbox.io/s/react-photo-album-67-1-lmf9d2?file=/src/App.tsx

While I agree this seems to be a glitch with CodeSandbox, there does seem to be a way to make CodeSandbox happy on Windows since I don't remember other links giving me this problem.  It seems to only be links from the [Examples](https://react-photo-album.com/examples) in React Photo Album.  If you don't have a Windows box handy and would like me to test a fix, I would be happy to do that.

**Reply by igordanchenko** [reply, marked as answer]
The difference is that `https://codesandbox.io/s/` are legacy [browser sandboxes](https://codesandbox.io/docs/learn/sandboxes/overview?tab=browser) while `https://codesandbox.io/p/sandbox/` are modern [cloud sandboxes](https://codesandbox.io/docs/learn/sandboxes/overview?tab=cloud).

Some other users are experiencing the same CodeSandbox error as you do. Please follow https://github.com/codesandbox/codesandbox-client/issues/7761 for updates. I don't believe there is anything I can do to resolve this issue.
